jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2820,7 +2820,7 @@ public boolean isAnnotationPresent(Class<? extends Annotation> annotation) {
 
 /**
  * Cast this Class to a subclass of the specified Class. 
- * 
+ * @param <U> the type to cast this class object to
```

Comment:
There shouldn't be two different @param statements for the single parameter.

Reply:
Agree I will fix that. thanks